src/group_gui.cpp

```diff
@@ -253,6 +271,12 @@ class VehicleGroupWindow : public BaseVehicleListWindow {
 		const GroupStatistics &stats = GroupStatistics::Get(this->vli.company, g_id, this->vli.vtype);
 		bool rtl = _current_text_dir == TD_RTL;
 
+		/* draw fold / unfold button */
+		int x = rtl ? right - WD_FRAMERECT_RIGHT - 8 - this->column_size[VGC_FOLD].width + 1 : left + WD_FRAMERECT_LEFT + 8;
+		if (has_children) {
+			DrawSprite(Group::Get(g_id)->folded ? SPR_CIRCLE_FOLDED : SPR_CIRCLE_UNFOLDED, PAL_NONE, rtl ? x - indent * LEVEL_WIDTH : x + indent * LEVEL_WIDTH, y + (this->tiny_step_height - this->column_size[VGC_FOLD].height) / 2);
```

Comment:
indent level has already been multiplied by LEVEL_WIDTH by the caller of DrawGroupInfo, hence the display issue.